src/utils/network/utils.js

```diff
@@ -244,6 +244,30 @@ export function buildURL({ AMaaSClass, AMId, resourceId, stage, apiVersion }) {
         apiVersion
       })}/transaction/aggregate_pnls`
       break
+    case 'eod':
+      baseURL = `${getEndpoint({
+        stage,
+        apiVersion
+      })}/marketdata/eod-prices`
+        break
+    case 'curve':
+      baseURL = `${getEndpoint({
+        stage,
+        apiVersion
+      })}/marketdata/curves`
+      break
+    case 'fxRate':
```

Comment:
Elsewhere you have defined this as `fxrate` so this case will never be matched.

Reply:
Updated this in the other file.